### PR TITLE
Issue #221 by agreazel: Use Backdrop branding for the installer.

### PR DESCRIPTION
**Pull request by jenlampton**
New PR for https://github.com/backdrop/backdrop-issues/issues/221

### CODE REVIEW

File: core/themes/seven/install.css [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    padding-left: 30px; /* LTR */
+    width: 65%;
+  }
+}
+body.install-page #page-title {
+  background: transparent;
+  padding-top: 0;
+}
+
+@media all and  (max-width: 48em) { /* 768px */
```
Comment (quicksketch) [inline core/themes/seven/install.css]: Let's just change this to px straight-up. The code comment isn't accurate if the body font gets changed. There's an extra space before `(max-width:`.

File: core/themes/seven/install.css [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    padding: 10px 20px 0;
+  }
+}
+body.install-page .task-list {
+  margin-left: 0; /* LTR */
+  list-style-type: none;
+  list-style-image: none;
+  padding-left: 0; /* LTR */
+  padding-bottom: 1em;
+}
```
Comment (quicksketch) [inline core/themes/seven/install.css]: LTR styles are specified here, but they're not provided in a `[dir="rtl"]` selector.

File: core/themes/seven/install.css [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+.install-page .password-parent {
+  width: auto;
+}
+.install-page div.form-item div.password-suggestions {
+  float: none;
+  width: auto;
+}
+
+@media all and (max-width: 975px) and (min-width: 48em) {
```
Comment (quicksketch) [inline core/themes/seven/install.css]: Let's stick with px everywhere, these kinds of mixings can be confusing.

File: core/themes/seven/install.css [hunk 4]
```diff
@@ -0,0 +1,171 @@
+
+/**
+ * @file
+ * Css for the install, maintenance, and update page.
```
Comment (quicksketch) [inline core/themes/seven/install.css]: Css => CSS

File: core/themes/seven/template.php [hunk 5]
```diff
@@ -9,6 +9,8 @@ function seven_preprocess_maintenance_page(&$vars) {
   // maintenance-page.tpl.php template. So, to have what's done in
   // seven_preprocess_html() also happen on the maintenance page, it has to be
   // called here.
+  $path = backdrop_get_path('theme', 'seven');
+  backdrop_add_css($path . '/install.css');
```
Comment (quicksketch) [inline core/themes/seven/template.php]: Shouldn't this only be added during the install process? Looks like it's going to be added all the time.

**Comment by quicksketch** [comment]
Couple of areas need some work here. In particular let's try to be consistent with our media queries and be sure the new changes work in RTL languages. There are some additions here that need new RTL rules.

**Comment by jenlampton** [comment]
Okay, RTL support in place.  CSS cleaned up, pixelz used for media queries, and maintenance pages look sexy now too. retesting.

**Pull request merged**